devtools/conda-recipe/meta.yaml

```diff
@@ -23,6 +23,7 @@ requirements:
     - parmed
     - mdtraj
     - foyer
+    - openmm
```

Comment:
Good catch. I was wondering at first why this change was necessary, but looking into it we do use OpenMM in [`compound.py`](https://github.com/mosdef-hub/mbuild/blob/master/mbuild/compound.py#L19) (specifically in [`energy_minimization`](https://github.com/mosdef-hub/mbuild/blob/master/mbuild/compound.py#L1233) after #334). We can easily modify this function to use ParmEd or MDTraj so that we don't require OpenMM as a dependency. I'll get this fixed shortly.